econml/dml.py

```diff
@@ -1162,6 +1176,13 @@ def fit(self, Y, T, X=None, W=None, *, sample_weight=None, sample_var=None, grou
                            sample_weight=sample_weight, sample_var=None, groups=groups,
                            inference=inference)
 
+    def shap_values(self, X, *, feature_names=None, treatment_names=None, output_names=None):
+        model = copy.deepcopy(self.model_cate)
+        model.__class__ = RandomForestRegressor
```

Comment:
This looks hacky - at least add a comment explaining why it works.

Reply:
Done.